lib/src/realtime/src/realtime_channel_interface.dart

```diff
@@ -92,6 +94,10 @@ abstract class RealtimeChannelInterface
   /// Warning: the name/ names are not channel names, but message names.
   /// See [Message.dart] for more information.
   ///
+  /// Calling subscribe the first time on a channel will automatically attach
+  /// channel. If a channel is detached, subscribing again will not reattach
```

Comment:
```suggestion
  /// that channel. If a channel is detached, subscribing again will not reattach
```

(though I'm surprised this commentary addition falls within the scope of work being done for this pull request)